g/gojsonschema/gojsonschema_rhel_8.3.sh

```diff
@@ -0,0 +1,39 @@
+# -----------------------------------------------------------------------------
+#
+# Package       : gojsonschema
+# Version       : v1.2.0
+# Source repo   : https://github.com/xeipuuv/gojsonschema.git
+# Tested on     : RHEL 8.3
```

Comment:
have we tested this on a UBI based container environment as well?

Reply:
I have tested on ubi 8.3 so i will change 'Tested on' and file name wilt 'rhel 8.3'